### PR TITLE
Documentation - Linear Gauge

### PR DESCRIPTION
**Pull request by svdimitr**
## New Component Checklist (delete for other types of PRs)

- [x] - Overview article
    - [x] General information on the purpose of the component.
    - [x] Instruction on how to add the component to a razor component.
    - [x] Basic configuration of the component - code snippet with a scenario that is both basic, and meaningful.
    - [x] Provide information on the component's reference.
    - [x] List the available features for the component and/or provide links (slugs) for the articles that explain them in details. For simpler components listing parameters might be enough, for more complex ones sections with a sentence or two that links corresponding articles might be more suitable.
    - [x] Maybe some Example(s) - very common scenarios, notes, typical use case, etc. Not mandatory, depends on the component.

- [ ] - Data binding article - applicable if data can be provided to the component. My be one article or a folder with several articles (examples - DropDownList and TreeView)

- [x] - Feature/component specific articles - those articles explain the component specific features, which do not suit in any of the before-mentioned articles. Examples of such articles are the [Navigator](https://docs.telerik.com/blazor-ui/components/stockchart/navigator) for the Stock Chart, the [Indeterminate](https://docs.telerik.com/blazor-ui/components/checkbox/indeterminate-state) state for the CheckBox and others. 

- [ ] - Selection article - applicable if the component exposes items selection. May be a folder.
    [ ] - Single item selection
    [ ] - Multiple item selection

- [ ] - Templates - applicable if additional customization is allowed with Templates. Usually a folder, see the grid 

- [ ] - Events article - applicable if the component expose events. 
Examples of such events could be:
 * OnClick
 * OnChange
 * ValueChanged
 * SelectedItemsChanged
 * OnRead
 * etc.
 
- [ ] Update Input Validation, Globalization, Keyboard Navigation, Accessibility Compliance articles as applicable.

- [x] Link from the Overview components list

### CODE REVIEW

**Review by marin-bratanov — APPROVED** [review]
It looks like a lot of comments, but nothing is really major, just minor tweaks that can take the whole docs to a new level. I like the structure and how you can find the main elements of the component at the root level of the docs, and so many details and examples for each.
While I'd like to see those comments acted on, I am approving this PR.
EDIT: you could also mention the gauges (may be as a single bullet "Gauges") in the KBD nav article just like we do for the chart so it is clear they can't have kbd navigation.

File: _config.yml [hunk 1]
```diff
@@ -427,6 +429,10 @@ intro_columns:
       "Radar Column Charts": "chart-types-radarcolumn"
       "Radar Line Charts": "chart-types-radarline"
       "Stock Chart": "stockchart-overview"
+      "Gauge - Linear": "linear-gauge-overview"
```
[the remark this answers is not in the document]
Comment (svdimitr) [reply _config.yml:432]: Done

File: components/gauge-linear/overview.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+![Basic Linear Gauge](images/basic-linear-gauge.png)
+
+````CSHTML
+@* Setup a basic linear gauge *@
+
+<TelerikLinearGauge>
+    <LinearGaugePointers>
+        <LinearGaugePointer Value="10" />
+
+        <LinearGaugePointer Value="20" />
```
[the remark this answers is not in the document]
Comment (svdimitr) [reply components/gauge-linear/overview.md:44]: This is the way it renders, when zooming more closely you could see that there is a difference (slight, but different).

File: components/gauge-linear/pointers.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        <LinearGaugePointer Value="30" Color="red" Shape="@LinearGaugePointerShape.Arrow">
+        </LinearGaugePointer>
+        
+    </LinearGaugePointers>
+</TelerikLinearGauge>
+````
+
+## Size
+
+The `Size` (`double`) parameter controls the size of the pointers.
```
[the remark this answers is not in the document]
Comment (svdimitr) [reply components/gauge-linear/pointers.md:109]: It works with Bar Indicator too

**Pull request merged**